week-2-practise/level-2/4-slice.js

```diff
@@ -3,10 +3,16 @@
 // - NOT change the original array
 // - return a new array with the item removed
 // - remove the item at the specified index
+function firstPartOfArr(arr, index) {
+  return arr.slice(0, index);
+}
+
+function lastPartOfArr(arr, index) {
+  return arr.slice(index + 1, arr.length);
+}
 
 function remove(arr, index) {
-  let newArr = arr.slice(0, index).concat(arr.slice(index + 1, arr.length));
-  return newArr; // complete this statement
+  return firstPartOfArr(arr, index).concat(lastPartOfArr(arr, index)); // complete this statement
```

Comment:
clever use of functions in one line well Done !

Reply:
> Well done. Can we do this by using "arr.splice()"?

As the instructions stated "NOT change the original array", I used ```.slice()``` method. You can get the same result with the ```.splice()``` method as well. But your original array will be changed.